src/vnsw/agent/oper/interface.cc

```diff
@@ -559,7 +559,17 @@ MacAddress GetMacAddressFromLuid(const NET_LUID intf_luid) {
     }
 
     LOG(ERROR, "mac address not found for LUID " << intf_luid.Value);
-    assert(0);
+    return boost::none;
+}
+
+static std::string LuidToString(const NET_LUID intf_luid) {
+    std::stringstream ss;
+
+    ss << "<IfType = " << intf_luid.Info.IfType
+       << ", NetLuidIndex = " << intf_luid.Info.NetLuidIndex
+       << ", Value = " << intf_luid.Value << ">";
+
+    return ss.str();
 }
 #endif
```

Comment:
How big is this #ifdef? Don't you want to move this to another file (Windows only)?

Reply:
I think that is a good idea. I have created a task in Jira (JW-1107) for this, so it can be done later.